ui/desktop/app/controllers/scopes/scope/projects/targets/index.js

```diff
@@ -76,9 +82,11 @@ export default class ScopesScopeProjectsTargetsIndexController extends Controlle
             proxy_address: address,
             proxy_port: port,
             target_id: target.id,
+            expiration_time: expiration,
```

Comment:
[If you fix the comment above](https://github.com/hashicorp/boundary-ui/pull/2018/files#r1417747186), this should be OK, Since from the CLI we are receiving `expiration`, but as you can see, here we are pushing to our store (UI) and there, we have model `expiration` as `expiration_time`.